railties/lib/rails/generators/app_base.rb

```diff
@@ -312,7 +319,7 @@ def assets_gemfile_entry
       def webpacker_gemfile_entry
         return [] if options[:skip_javascript]
 
-        if options.dev? || options.edge?
+        if options.dev? || options.edge? || options.master?
```

Comment:
Think there's a separate PR in extracting this? We probably aren't likely to change this now, but still think it makes sense to make the code more readable. Maybe `pull_from_github?` would do?

Reply:
Oooh good point. I love extracting compound conditionals. After I wrap up #38761 I'll definitely have a dig at that :)